Update new-asset-proposal.md (#588)
vegaprotocol/documentation: docs/tutorials/proposals/new-asset-proposal.md

If the vote passes, the network validators will then enable the asset on the [bridge contract](../../api/bridge/contracts/ERC20_Bridge_Logic#tag/TradingDataService/operation/TradingDataService1_ERC20WithdrawalApproval) which will enable deposits and withdrawals for that token.

## Anatomy of a new asset proposal
The key inputs on a new asset proposal are as follows:

| Field | Description | Example |
| ----------- | ----------- | ----------- |
| `name` | Name of the asset (string) | Testnet DAI |
| `symbol` | Symbol of the asset (string)  | tDAI |
| `decimals` | Number of decimal / precision handled by this asset (string)  | 18 |
| `quantum` | The minimum economically meaningful amount in the asset (string).  This should be the amount of the asset roughly equal to 1 USD.  It is used in a number of ways by the protocol but only requires precision to an order of magnitude level.  eg. for BTC, one BTC = 26,583 USD today; in this case a quantum of 1 / 25,000 or 0.00004 is sufficient, converted to asset decimals so 40000000000000. | 1000000000000000000 |

> | `withdrawThreshold` | The maximum you can withdraw instantly in asset decimals. All withdrawals over the threshold will be delayed by the withdrawal delay which will be set to 24h during Alpha Mainnet.  It is advised during Alpha Mainnet to set this to 0 so that all withdrawals are subject to the delay. | 0 |

Reviewer: ```suggestion
| `withdrawThreshold` | The maximum you can withdraw instantly in asset decimals. All withdrawals over the threshold will be delayed by the withdrawal delay, which can be seen on the ERC-20 bridge per asset. Setting this to 0 means all withdrawals will be subject to the delay. | 0 |
```

> | `lifetimeLimit` | The lifetime deposit limit per address in asset decimals.  These limits exist to discourage retail users from placing more funds than they can afford to lose on the network.  It is advised that this be set to no more 10,000 USD equivalent during Alpha Mainnet.  Users are able to opt out of this functionality using the `exempty_depositor` write function on the ERC20 contract if they wish to.   | 10000000000000000000000 |

Reviewer: ```suggestion
| `lifetimeLimit` | The lifetime deposit limit per address in asset decimals. Users are able to opt out of this functionality using the `exempty_depositor` write function on the ERC20 contract if they wish to. Suggested value: equivalent of 10,000 USD | 10000000000000000000000 |
```

This isn't alpha mainnet docs
